src/_includes/soft_mod_cheat_sheet_cta_banner.html

```diff
@@ -0,0 +1,30 @@
+<div class="container section g-pt-200 g-pb-100">
```

Comment:
It would be nice to use a single BEM style block classname for these styles.

As the file is called `soft_mod_cheat_sheet_cta_banner` that would seem to be the block name. Is that a name that @anguspaterson is using in the designs? Happy to look at it together sometime